src/Mhwd.cpp

```diff
@@ -614,7 +620,7 @@ bool Mhwd::runScript(std::shared_ptr<Config> config, MHWD::TRANSACTIONTYPE opera
     for (auto&& foundDevice = foundDevices.begin();
             foundDevice != foundDevices.end(); ++foundDevice)
     {
-        bool found = false;
+        static bool found = false;
```

Comment:
Because the variable "found" is declared inside the for loop and this way, declared as static, is just declared once and used many times instead of declared multiples times when the loop is running. It's also an optimization for saving memory.